app/views/content/is-teaching-right-for-me/maths/_article.html.erb

```diff
@@ -0,0 +1,92 @@
+
+<p>It's also our government's goal for all young people to study maths to age 18, equipping them with the skills they need for life.</p>
+    <% end %>
+  </section>
+
+  <section class="col col-720 col-horizontal col-space-m">
+    <div>
+      <h2>What you'll be teaching</h2>
+      <p>You'll teach the national curriculum. Themes you’ll cover for 11 to 16 year olds (key stage 3 and 4):</p>
+      <ul class="vertical-tags">
+        <li class="tag">Numbers</li>
+        <li class="tag">Algebra</li>
+        <li class="tag">Ratio, proportion and rates of change</li>
+        <li class="tag">Geometry and measures</li>
+        <li class="tag">Probability and statistics</li>
+      </ul>
+
+      <p>Themes you’ll cover for 16 to 18 year olds (key stage 5) - A or AS level core maths:</p>
+      <ul class="vertical-tags">
+        <li class="tag">Analysis of data</li>
+        <li class="tag">Maths for personal finance</li>
+        <li class="tag">Estimation</li>
+        <li class="tag">Critical analysis of given data and models</li>
+      </ul>
+    <p><a href="https://www.gov.uk/government/publications/national-curriculum-in-england-mathematics-programmes-of-study/national-curriculum-in-england-mathematics-programmes-of-study#key-stage-3">Read the full national curriculum for more information about teaching maths in secondary schools</a>.</p>
+    </div>
+</section>
+<section class="col col-845 inset">
+    <%= render Content::PhotoQuoteListComponent.new(
+      [
+        {
+          image: "static/images/content/subjects/rotna-roy.jpeg",
+          heading: "Empowering my pupils",
+          text: "I enjoy showing my pupils that maths is essential in everyday life. Whether it is identifying patterns, interpreting data, managing finances, or understanding the spaces and structures around us, everything uses maths.",
+          accreditation: "Rotna Roy, Maths Lead at a school in London",
+        }
+      ],
+      colors: %w[yellow],
+      numbered: false
+    ) %>
+  </section>
+
+<section class="col col-space-m col-720">
+<h2>Check your qualifications</h2>
+<p>You'll need a bachelors degree before training to teach, but it does not have to be in maths.</p>
+<p>Maths teacher training courses have had successful applications from candidates with a range of degrees, including psychology, music, architecture and law.</p>
+<p>Your training provider may need you to brush up on your computing knowledge before you become a maths teacher. You can do a <a href="/how-to-apply-for-teacher-training/subject-knowledge-enhancement">subject knowledge enhancement (SKE) course</a> before you start your teacher training to top up your subject knowledge. </p>
+<p><a href="/is-teaching-right-for-me/qualifications-you-need-to-teach">Find out more about the qualifications needed to teach</a>.</p>
+<p><a href="/non-uk-teachers">Non-UK citizens can find out more about training to teach in England</a>.</p>
```

Comment:
@caitlin-reidy this link takes them to the international category page. But I think it would be better to take them to the page that includes qualifications info. Not sure if you may have just put the wrong link in. https://getintoteaching.education.gov.uk/non-uk-teachers/train-to-teach-in-england-as-an-international-student

Laura's working on the new qualifications page so once that's live we can always change the link to take people there instead as it's quite easy to miss the info on this page. And from the existing link/ title page it's not clear it's about checking qualifications.